frontend/src/components/tracks/index-pm.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+import Header from './header/index';
+import Footer from '../shared/footer/index';
+import TrackImage from './track-image/index';
+import WhatYouLearn from './what-you-learn/index';
+import OfferResponsibilities from './offer-responsibilities/index';
+import Expectations from './expectations/index';
+import { or, header, expectations } from './data/pm';
+
+/**
+ * The Apply Now button is not abstracted away in its own component, because it's temporary there
+ * TrackImage is for now the same for all tracks. It can be later made more flexible if track heads
+ * decide to use different images.
+ */
+class TrackPM extends React.Component {
```

Comment:
Not sure about the name of this class here. Perhaps I misunderstand the code though. ;) 
I had the feeling, you want to put the track specific stuff in the consts and use this class as a template for all. 

Hence I'd go for a more generic class name like `TrackTemplate` or something like that.

Reply:
So I was thinking there are two approaches here:

The first one is index-pm, index-ixd etc for each track that are referenced here:

https://github.com/DigitalProductschool/website/pull/50/files#diff-adfbb4228db94487c25e2b492a56ded1R25 

The second option as you propose a generic template which catches all routes and then based on routing parameters we select the track. The advantage is less duplication, but more if statements (for instance, the apply button, the data, etc...).

I think both ok, don't have strong preferences here

Reply:
Ah okay. Then I misunderstood the comments above. 😊 
I thought you meant to have it as a template class, but was confused with the name afterwards. ;) 

I'd prefer to duplicate the code, as every track head is allowed to customize their track page.